portal/src/components/Materials/MaterialInfo/MaterialInfo.component.js

```diff
@@ -61,9 +61,15 @@ export default {
      */
     closePopupSaveRating() {
       this.isShow = false;
-      this.rating_given = true;
     },
-
+    /**
+     * Check in sessionStorage if material has been rated by the current user"
+     * @param external_id of material - String
+     */
+    isMaterialRated(materialId) {
+      const ratings = JSON.parse(sessionStorage.getItem('ratedMaterials')) || []
```

Comment:
JSON.parse(null) lijkt me een risico. Weten we zeker dat alle browsers daar niet "null" van maken?